docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,328 @@
+# High Availability
```

Comment:
Might want to add a "summary" section that outlines the intended reader and scope.  More important what this document "is not"

Reply:
Agree! "scope" and "non-goal" sections needed here